packages/react-query/package.json

```diff
@@ -39,8 +39,13 @@
   "types": "dist/index.d.ts",
   "files": [
     "dist",
-    "src"
+    "src",
+    "bin",
+    "scripts"
```

Comment:
When trying to change to a TypeScript file, I think it is possible to do all the work in src and put all bin and scripts files in dist.

```suggestion
```